auditlog_tests/tests.py

```diff
@@ -310,6 +311,18 @@ def test_register_mapping_fields(self):
         )
 
 
+class SimpeMaskedFieldsModelTest(TestCase):
+    """Log masked changes for fields in mask_fields"""
+
+    def test_register_mask_fields(self):
+        smm = SimpleMaskedModel(address="Sensitive data", text="Looong text")
+        smm.save()
+        self.assertTrue(
+            smm.history.latest().changes_dict["address"][1] == "*******ve data",
+            msg="The diff function masks 'address' field.",
+        )
```

Comment:
Let's use `assertEqual` here:
```suggestion
        self.assertEqual(
            smm.history.latest().changes_dict["address"][1],
            "*******ve data",
            msg="The diff function masks 'address' field.",
        )
```